examples/011_pymechanical_solid_workflow.py

```diff
@@ -0,0 +1,390 @@
+# ------------------------------------------------
+#
+# Import geometry, mesh, and named selections into Mechanical
+
+pyacp.mechanical_integration_helpers.import_acp_solid_model(
+    mechanical=mechanical_2, cdb_path=working_dir_path / solid_model_cdb_file
+)
+
+
+# %%
+# Import materials into Mechanical
+
+mechanical_2.run_python_script(f"Model.Materials.Import({str(working_dir_path / matml_file)!r})")
+
+# %%
+# Import plies into Mechanical
+
+pyacp.mechanical_integration_helpers.import_acp_composite_definitions(
+    mechanical=mechanical_2, path=working_dir_path / solid_model_composite_definitions_h5
+)
+
+
+# %%
+# Set boundary condition and solve
+# ---------------------------------
+#
+# Set boundary condition and solve
+
+mechanical_2.run_python_script(
+    textwrap.dedent(
+        """\
+        analysis = Model.AddStaticStructuralAnalysis()
+
+        front_face = Model.AddNamedSelection()
+        front_face.Name = "front_face"
+        front_face.ScopingMethod = GeometryDefineByType.Worksheet
+        front_face.GenerationCriteria.Add(None)
+        front_face.GenerationCriteria[0].EntityType = SelectionType.GeoFace
+        front_face.GenerationCriteria[0].Criterion = SelectionCriterionType.LocationX
+        front_face.GenerationCriteria[0].Operator = SelectionOperatorType.Largest
+        front_face.Generate()
+
+        back_face = Model.AddNamedSelection()
+        back_face.Name = "back_face"
+        back_face.ScopingMethod = GeometryDefineByType.Worksheet
+        back_face.GenerationCriteria.Add(None)
+        back_face.GenerationCriteria[0].EntityType = SelectionType.GeoFace
+        back_face.GenerationCriteria[0].Criterion = SelectionCriterionType.LocationX
+        back_face.GenerationCriteria[0].Operator = SelectionOperatorType.Smallest
+        back_face.Generate()
+
+        fixed_support = analysis.AddFixedSupport()
+        fixed_support.Location = back_face
+
+        force = analysis.AddForce()
+        force.DefineBy = LoadDefineBy.Components
+        force.XComponent.Output.SetDiscreteValue(0, Quantity(1e5, "N"))
+        force.Location = front_face
+
+        analysis.Solve(True)
+        """
+    )
+)
+
+rst_file = [filename for filename in mechanical_2.list_files() if filename.endswith(".rst")][0]
+matml_out = [filename for filename in mechanical_2.list_files() if filename.endswith("MatML.xml")][
+    0
+]
+
+# %%
+# Postprocess results
+# -------------------
+#
+# Evaluate the failure criteria using the PyDPF Composites.
+
+max_strain = pydpf_composites.failure_criteria.MaxStrainCriterion()
+cfc = pydpf_composites.failure_criteria.CombinedFailureCriterion(
+    name="Combined Failure Criterion",
+    failure_criteria=[max_strain],
+)
+
+composite_model = pydpf_composites.composite_model.CompositeModel(
+    composite_files=pydpf_composites.data_sources.ContinuousFiberCompositesFiles(
+        rst=rst_file,
+        composite={
+            "shell": pydpf_composites.data_sources.CompositeDefinitionFiles(
```

Comment:
I think `shell` is not needed since only the solid model was passed to the downstream analysis. All required data are stored in solid_model_composite_definitions_h5.